invenio_oaiserver/response.py

```diff
@@ -257,7 +254,10 @@ def listidentifiers(**kwargs):
 
 def listrecords(**kwargs):
     """Create OAI-PMH response for verb ListIdentifiers."""
-    from invenio_records.models import RecordMetadata
+    kwargs['verb'] = 'ListRecords'
```

Comment:
Ok, but why we need also to pass the verb if we already know what we want by calling `listrecords()`?
I mean, imagine if the function will be reused somewhere. 
It a point of possible mistake because you can call `listrecords()` but pass the verb `ListSets`.